mne/io/kit/kit.py

```diff
@@ -760,6 +760,7 @@ def get_kit_info(rawfile, allow_unknown_format, standardize_names=None,
     is_exg = [ch['type'] in (KIT.CHANNEL_EEG, KIT.CHANNEL_ECG)
               for ch in channels]
     exg_gains /= 2 ** (adc_stored - 14)
+    exg_gains[exg_gains == 0] = ad_to_volt
```

Comment:
FYI with this change I get 2.55 V on that channel at that time, feel free to try it

Reply:
Thank you for finding the solution, I tried it, it is correct.